mlx_graphs/nn/conv/gin_conv.py

```diff
@@ -88,13 +111,19 @@ def __call__(
         if isinstance(node_features, mx.array):
             node_features = (node_features, node_features)
 
+        if 'edge_projection' in self:
+            edge_features = self.edge_projection(edge_features)
+        
         dst_features = node_features[1]
 
         aggr_features = self.propagate(
             edge_index=edge_index,
             node_features=node_features,
-            message_kwargs={"edge_weights": edge_weights},
+            message_kwargs={"edge_weights": edge_weights,
+                            "edge_features": edge_features},
         )
+        if 'edge_projection' in self:
+            aggr_features += edge_features
```

Comment:
You can't sum node embeddings with edge embeddings here, you have to do that in the `message` function.